public/app/core/components/sidemenu/SideMenu.tsx

```diff
@@ -24,7 +24,7 @@ export const SideMenu: FC = React.memo(() => {
   }
 
   return (
-    <nav className="sidemenu" data-testid="sidemenu">
+    <nav className="sidemenu" data-testid="sidemenu" aria-label="Main menu">
       <a href={homeUrl} className="sidemenu__logo" key="logo">
         <Branding.MenuLogo />
       </a>
```

Comment:
Any particular reason for not having the BottomSection in the nav element? is part of the same navigation no? or am I missing something related to a11y?

Reply:
Bottom section does not really contain navigation items. It could possibly be the profile, but not the help button. It's probably more a question of taste here. I'm fine with you skipping it.